fapi-client/src/main/scala/com/gu/facia/api/utils/ItemKicker.scala

```diff
@@ -90,14 +90,22 @@ object ItemKicker {
     case _ => None
   }
 
+  def stringIfPlainText(string: String) : Option[String] = {
+    val StringWithHtml = "<\\w+>".r.unanchored
```

Comment:
scala.xml.XML.loadString("foo")
org.xml.sax.SAXParseException; lineNumber: 1; columnNumber: 1; Content is not allowed in prolog.

scala> scala.xml.XML.loadString("<p> foo <br> bar </p>")
org.xml.sax.SAXParseException; lineNumber: 1; columnNumber: 20; The element type "br" must be terminated by the matching end-tag "</br>".